viewer/packages/pointclouds/src/styling/AnnotationListPointCloudObjectCollection.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { StylableObjectCollection } from './PointCloudObjectCollection';
+
+/**
+ * A simple PointCloudObjectCollection that consists of an explicitly provided list of annotation IDs
+ */
+export class AnnotationListStylableObjectCollection extends StylableObjectCollection {
```

Comment:
See if you can align naming here with naming we have for the corresponding class for CAD

Reply:
Renamed to `AnnotationIdPointCloudObjectCollection`